### ISSUE TITLE
encountered this error: Report E2BIG spawn E2BIG

### ISSUE THREAD

**Issue by gpake** [vknabel/vscode-swiftlint#9]
every time I launch the VSCode it shows but looks the lint works for me.

```
Error: spawn E2BIG
	at ChildProcess.spawn (internal/child_process.js:394:11)
	at spawn (child_process.js:553:9)
	at Object.execFile (child_process.js:237:17)
	at Object.module.<computed> [as execFile] (electron/js2c/asar.js:163:31)
	at /Users/gpake/.vscode/extensions/vknabel.vscode-swiftlint-1.0.5/out/lint.js:129:38
	at new Promise (<anonymous>)
	at execSwiftlint (/Users/gpake/.vscode/extensions/vknabel.vscode-swiftlint-1.0.5/out/lint.js:128:12)
	at Object.<anonymous> (/Users/gpake/.vscode/extensions/vknabel.vscode-swiftlint-1.0.5/out/lint.js:47:38)
	at Generator.next (<anonymous>)
	at fulfilled (/Users/gpake/.vscode/extensions/vknabel.vscode-swiftlint-1.0.5/out/lint.js:5:58)
```

**Comment by vknabel** [comment]
Thanks for the report!
I will have a deeper on this the next few days.

**Comment by vknabel** [comment]
@gpake The fix has been released in [1.1.0](https://github.com/vknabel/vscode-swiftlint/releases/tag/1.1.0)!
